cluster/test-smoke.sh

```diff
@@ -23,10 +23,10 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 
-TEST_ARGS="$@"
+TEST_ARGS="$*"
 
 SMOKE_TEST_FOCUS_REGEX="Guestbook.application"
 
-exec "${KUBE_ROOT}/cluster/test-e2e.sh" -ginkgo.focus="${SMOKE_TEST_FOCUS_REGEX}" ${TEST_ARGS}
+exec "${KUBE_ROOT}/cluster/test-e2e.sh" -ginkgo.focus="${SMOKE_TEST_FOCUS_REGEX}" "${TEST_ARGS}"
```

Comment:
```
In ./cluster/test-smoke.sh line 32:
exec "${KUBE_ROOT}/cluster/test-e2e.sh" -ginkgo.focus="${SMOKE_TEST_FOCUS_REGEX}" ${TEST_ARGS}
                                                                                  ^-- SC2086: Double quote to prevent globbing and word splitting.
```

Reply:
I'm not sure if this is equivalent behavior

Reply:
As you said, it confused me too.  :(
I can only give up the modification of it for the time being.
I have rolled back my changes to this file.